TypeCobol/Analysis/AnalyzerProvider.cs

```diff
@@ -10,103 +12,177 @@ namespace TypeCobol.Analysis
     /// <summary>
     /// IAnalyzerProvider default implementation. Can be used directly by adding activation delegates
     /// with the appropriate AddActivator method or by subclassing it.
+    /// It can also be used to allow composition of multiple providers into one.
+    /// This ensures that the creation of IAnalyzers won't interrupt the parsing.
```

Comment:
'This class also ensures', it's not related to the previous sentence about composition of providers.